source/jormungandr/tests/routing_tests_experimental.py

```diff
@@ -660,6 +660,53 @@ def test_min_taxi(self):
         taxi_fallback = next((j for j in response['journeys'] if "taxi" in j['tags']), None)
         assert 'deleted_because_too_short_heavy_mode_fallback' in taxi_fallback['tags']
 
+    def test_max_taxi_duration_to_pt(self):
```

Comment:
Comments about what's happening in each step of the test would be welcome to understand it! :wink: